### PR TITLE
Tests,editing of event functionality and some appearence changes

### PR DESCRIPTION
**Pull request by AlicePru**
### Checklist
- [x] I've run `mvn test` from the root directory to see all new and existing tests pass
- [x] I've followed code style and run and ensured the code style is valid
- [x] I've created new tests if necessary.

### Motivation and Context
#20
#8 
#10 
#13 
#31 

### Description
Were done tests for almost all GET and POST methods in EventOrganizationApi class except test of function,where we get as parameter JsonObject. Also was added functional for editing event by reusing of Add Page , added oportunity for going to eventPage by clicking on event from MyEvent page and from start page. A little bit changed style of pages

### CODE REVIEW

File: src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java [hunk 1]
```diff
@@ -53,31 +50,69 @@ public TopicListDto getAllOpenEvents() {
     @Path("/save")
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java]: Create separate methods for save and update events.

File: src/main/java/lv/ctco/javaschool/eventorganaizer/entity/EventListDto.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package lv.ctco.javaschool.eventorganaizer.entity;
+
+import java.util.List;
+
+public class EventListDto {
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/eventorganaizer/entity/EventListDto.java]: Do you need this class ? Is it possible not to wrap List into class ?

File: src/main/java/lv/ctco/javaschool/eventorganaizer/entity/TopicDto.java [hunk 3]
```diff
@@ -5,28 +5,66 @@
...
+    private Long id;
 
-    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool) {
+    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool, Long id) {
         this.topicName = topicName;
         this.topicAuthor = topicAuthor;
         this.date = date;
-        this.isFeedbackPool = isFeedbackPool;
+        if (isFeedbackPool) {
+            this.path = "pool.jsp";
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/eventorganaizer/entity/TopicDto.java]: plain text should be declared in constants

File: src/main/java/lv/ctco/javaschool/eventorganaizer/entity/TopicDto.java [hunk 4]
```diff
@@ -5,28 +5,66 @@
...
-    private boolean isFeedbackPool;
+    private String path;
+    private Long id;
 
-    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool) {
+    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool, Long id) {
         this.topicName = topicName;
         this.topicAuthor = topicAuthor;
         this.date = date;
-        this.isFeedbackPool = isFeedbackPool;
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/eventorganaizer/entity/TopicDto.java]: Choose on where to go should be in api controller, not in DTO.

File: src/main/webapp/app/add-event.jsp [hunk 5]
```diff
@@ -2,47 +2,82 @@
...
+    <title>Add/Edit Event</title>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <link rel="stylesheet" href="/resources/demos/style.css">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 </head>
-<body>
+<body onload="checkFunction()">
 <style>
```
Comment (StepanovsRomans) [inline src/main/webapp/app/add-event.jsp]: Styles and javascripts should be in separate files

File: src/main/webapp/app/event.jsp [hunk 6]
```diff
@@ -1,20 +1,82 @@
...
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
-    <title>{{EventName}}</title>
+    <script src="https://www.w3schools.com/lib/w3.js"></script>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <title id="title">{{eventName}}</title>
 </head>
+<style>
```
Comment (StepanovsRomans) [inline src/main/webapp/app/event.jsp]: Styles and javascripts should be in separate files

File: src/main/webapp/app/my-events.jsp [hunk 7]
```diff
@@ -1,16 +1,89 @@
...
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 <html>
 <head>
     <title>My Events</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <script src="http://www.w3schools.com/lib/w3data.js"></script>
 </head>
-<body>
+<style>
```
Comment (StepanovsRomans) [inline src/main/webapp/app/my-events.jsp]: Styles and javascripts should be in separate files

File: src/main/webapp/app/start.jsp [hunk 8]
```diff
@@ -7,30 +7,42 @@
     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
     <style>
```
Comment (StepanovsRomans) [inline src/main/webapp/app/start.jsp]: Styles and javascripts should be in separate files

File: src/test/java/lv/ctco/javaschool/auth/control/UserStoreTest.java [hunk 9]
```diff
@@ -5,10 +5,16 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.function.Executable;
 import org.opentest4j.AssertionFailedError;
-
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsEqual.equalTo;
 import static org.junit.jupiter.api.Assertions.*;
 
 class UserStoreTest {
+    @Test
```
Comment (StepanovsRomans) [inline src/test/java/lv/ctco/javaschool/auth/control/UserStoreTest.java]: Unused method

File: src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java [hunk 10]
```diff
@@ -32,55 +28,58 @@
...
+    @GET
+    @RolesAllowed({"USER", "ADMIN"})
+    @Path("/{id}")
+    public EventDto getEventById(@PathParam("id") Long id) throws IllegalArgumentException {
+        Optional<Event> event = eventStore.getEventById(id);
+        if (event.isPresent()) {
+            Event e = event.get();
+            return new EventDto(e.getName(), e.getDescription(), e.getDate(), e.getId());
+        } else {
+            throw new IllegalArgumentException();
```
Comment (FLARS) [inline src/main/java/lv/ctco/javaschool/eventorganaizer/boundary/EventOrganizationApi.java]: I guess it's more appropriate to use  ``` throw new EntityNotFoundException() ```

**Pull request merged**